tdrs-frontend/src/components/Reports/Reports.test.js

```diff
@@ -312,13 +319,16 @@ describe('Reports', () => {
         },
       })
     })
-    expect(store.dispatch).toHaveBeenCalledTimes(10)
+    expect(store.dispatch).toHaveBeenCalledTimes(13)
 
     // There should be 4 more dispatches upon making the submission,
     // one request to /reports for each file
+    await waitFor(() =>
+      expect(getByText('Submit Data Files')).toBeInTheDocument()
+    )
     fireEvent.click(getByText('Submit Data Files'))
     await waitFor(() => getByRole('alert'))
-    expect(store.dispatch).toHaveBeenCalledTimes(14)
+    expect(store.dispatch).toHaveBeenCalledTimes(18)
```

Comment:
Why did these change? Seems odd

Reply:
I changed the order of event dispatches in order to be able to "reset" the input values without changing the redux state. basically for the tests that means there's one extra dispatch for each search input after the search button is clicked.